### ISSUE TITLE
Needs better a11y compliancy

### ISSUE THREAD

**Issue by thinkflo** [Alexeykhr/vue-stripe-menu#189]
First off, great work!  I always loved the stripe dropdowns and you did a bang up job implementing it.

Is there any way to make this more navigable via keyboard and hense more a11y compliant?  The main section seem accessible by tab order, but you cannot seem to navigate into the subsections via cursor key or tabs.  If you could add this, then you would make your plugin absolutely perfect.
